examples/js/loaders/FBXLoader.js

```diff
@@ -1651,6 +1651,21 @@
 
 		addAnimations( sceneGraph, animations );
 
+    /*
+      parse ambient color - if it's not set to black (default), create an ambient light
+      Note: no way to set intensity, FBX only provides a color
+    */
+		var ambientColor = FBXTree.GlobalSettings.properties.AmbientColor.value;
+		var r = ambientColor[ 0 ];
+		var g = ambientColor[ 1 ];
+		var b = ambientColor[ 2 ];
+		if ( r !== 0 || g !== 0 || b !== 0 ) {
+
+			var color = new THREE.Color( r, g, b );
+			sceneGraph.add( new THREE.AmbientLight( color ) );
+
+		}
+
 		return sceneGraph;
```

Comment:
It might be better to move this to a second function, as the parseScene function is getting pretty big at this point.